### DISCUSSION TITLE
How do I set up a queue when connecting？

### DISCUSSION THREAD

**Discussion by APEESONNAME66** [beltran/gohive#222] [Q&A]
How do I set up a queue when connecting？
I am using a user that does not have default queue permissions and cannot execute set mapred.job.queue.name
`config.HiveConfiguration = map[string]string{
	"mapred.job.queue.name": viper.GetString("hive.queue"),
}`
Is this right?
@beltran

**Comment by beltran** [comment]
Yes, those settings will become part of the session.
